.gitignore

```diff
@@ -4,4 +4,5 @@
 
 # Only apps should have lockfiles
 yarn.lock
+.vscode/
```

Comment:
IDE-specific files should go in one's global gitignore, not per project

Reply:
Oh that's a great point, I don't know if i have a `~/.gitignore` in my dotfiles